src/System.Net.Http.WinHttpHandler/src/System/Net/Http/CertificateHelper.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net.Http
+{
+    // TODO: Move to System.Net.Common as part of System.Private.Networking.dll refactor.
```

Comment:
@davidsh This comment is out of date. The file will eventually need to be moved to Common\System\Net and the namespace probably changed.